bagpipes/catalogue/fit_catalogue.py

```
@@ -97,10 +97,11 @@ class fit_catalogue(object):
 
     def __init__(self, IDs, fit_instructions, load_data, spectrum_exists=True,
                  photometry_exists=True, make_plots=False, cat_filt_list=None,
-                 vary_filt_list=False, redshifts=None, redshift_sigma=0.,
+                 vary_filt_list=False, redshifts=None, redshift_sigma=None,
                  run=".", analysis_function=None, time_calls=False,
                  n_posterior=500, full_catalogue=False, load_indices=None,
-                 index_list=None, track_backlog=False):
+                 index_list=None, track_backlog=False, spec_units="ergscma",
```

Ruff (0.14.5)

Unused method argument: `track_backlog`

(ARG002)